lib/cl/externals/bundle-info.sexp

```diff
@@ -0,0 +1,14 @@
+(:CREATION-TIME #A((20) BASE-CHAR . "2017-10-31T11:49:23Z") :REQUESTED-SYSTEMS
```

Comment:
Is this a file that was supposed to be checked in?  It looks like it is machine-specific.

Reply:
all things put in `externals` are checked in, so whole directory will be removed (this file included).